src/contracts/core/DelegationManager.sol

```diff
@@ -214,7 +231,7 @@ contract DelegationManager is Initializable, OwnableUpgradeable, Pausable, Deleg
      */
     function decreaseDelegatedShares(address staker, IStrategy[] calldata strategies, uint256[] calldata shares)
         external
-        onlyStrategyManager
+        onlyStrategyManagerOrEigenPodManager
```

Comment:
I would prefer adding a check if any of the strategies is beacon chain eth, then the sender must be eigenpodmanager

Reply:
> I would prefer adding a check if any of the strategies is beacon chain eth, then the sender must be eigenpodmanager

that kinda just feels more finnicky to me? I get the idea, feel like they arrive at the same place but this is just extra checking... let me know if this is something you feel strongly about, because my _slight_ preference is keeping things as-is here